core.go

```diff
@@ -745,6 +745,36 @@ func ExtractChannel(src Mat, dst Mat, coi int) {
 	C.Mat_ExtractChannel(src.p, dst.p, C.int(coi))
 }
 
+// FindNonZero returns the list of locations of non-zero pixels
+//
+// For further details, please see:
+// https://docs.opencv.org/master/d2/de8/group__core__array.html#gaed7df59a3539b4cc0fe5c9c8d7586190
+//
+func FindNonZero(src Mat, idx Mat) {
```

Comment:
It should be `FindNonZero(src, idx Mat)`

Reply:
well that is true for almost any existing function in the codebase.

i'd rather prefer not to make a breaking style change here

Reply:
start an issue / general discussion about it ?

Reply:
we could but that's how we write functions in Go. We need to fix the other functions if that's the case, but you can start fixing your stuff here :wink:

Reply:
ok, let's see, how it turns out ..